dub.json

```diff
@@ -87,8 +93,10 @@
 		{
 			"name": "cgi",
 			"description": "web server library with cgi, fastcgi, scgi, and embedded http server support",
-			"targetType": "sourceLibrary",
-			"sourceFiles": ["cgi.d"]
+			"targetType": "library",
+			"sourceFiles": ["cgi.d"],
+			"importPaths": ["."],
+			"dflags": ["-mv=arsd.cgi=cgi.d"]
```

Comment:
so without the version thing on the user end, you cannot use the fastcgi, scgi, and embedded_http modes.... hmmmm